speech_assets/sampleUtterances.txt

```diff
@@ -28,9 +28,9 @@ GeeMusicPlayIFLRadioIntent Play I'm feeling lucky station
 GeeMusicPlayIFLRadioIntent Play my radio station
 GeeMusicPlayIFLRadioIntent Play my station
 
-GeeMusicCurrentlyPlayingIntent What is the current song?
-GeeMusicCurrentlyPlayingIntent What is currently playing?
-GeeMusicCurrentlyPlayingIntent What is playing right now?
-GeeMusicCurrentlyPlayingIntent What song is this?
-GeeMusicCurrentlyPlayingIntent What is the name of this song?
-GeeMusicCurrentlyPlayingIntent What is this?
+GeeMusicCurrentlyPlayingIntent What is the current song
```

Comment:
ohh right! I forgot that I had fixed that in my dashboard but didn't update the text file. Glad you caught that.